_posts/intjnotinfp/2020-09-16-functional-turtle-intjnotinfp.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+author: intjnotinfp
+title: "intjnotinfp's (refactored) functional turtle exercise/ reflection"
+---
+
+* intjnotinfp's (refactored) functional turtle exercise/ reflection
+** Embedded Code for Purpose of Comparison
+
+Here's my original code - this was from our very first turtle exercise:
+<iframe src="https://trinket.io/embed/python/804e40cfa2" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+And here's that code with a few additions at the bottom, which served to add (1) plain function and (1) function with parameters to the original:
+<iframe src="https://trinket.io/embed/python/dd9c2fe124" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+I'll add just the few lines of code that changed below so that you can look at them as I describe my process.
+```#draw a border around the logo
```

Comment:
I think the code needs to be on a separate line from the backticks. No worries, but give a look once this is merged to see if it's the way you want.